README.md

```diff
@@ -4,9 +4,21 @@
 
 [SMS module](https://github.com/johnsonandjohnson/openmrs-distro-cfl/wiki/Admin-SMS) to enable track and sending SMS to OpenMRS users.
 
+[Messsages module](https://github.com/johnsonandjohnson/openmrs-distro-cfl/wiki/Admin-Messages) to enable configuration and sending Message services to OpenMRS users
```

Comment:
I'm not sure I understand "sending message services to OpenMRS users". Should this be "message sending services for OpenMRS users"?

```suggestion
[Messages module](https://github.com/johnsonandjohnson/openmrs-distro-cfl/wiki/Admin-Messages) to enable configuration and sending message services to OpenMRS users
```